core/src/create.jl

```diff
@@ -691,7 +704,7 @@ function User(db::DB, config::Config)::User
             first_row = time[first_row_idx]
             is_active = true
         else
-            @error "User node #$node_id data not in any table."
+            @error "User node $node_id data not in any table."
```

Comment:
I had another look at this. `node_id` comes from `node_ids` as the third output of the function `static_and_time_node_ids`, which is a vector with element type `Int` which you can see from the function signature:

```julia
function static_and_time_node_ids(
    db::DB,
    static::StructVector,
    time::StructVector,
    node_type::String,
)::Tuple{Set{Int}, Set{Int}, Vector{Int}, Vector{String}, Bool}
```

Thus adding the `#` here is needed for consistency. My mistake! This also indicates that there is no unit test for this error message.